lib/trackler/problem.rb

```diff
@@ -6,15 +6,16 @@ class Problem
     attr_reader :slug, :root
```

Comment:
Good catch. I don't want to change the existing API, even if it isn't used (or just isn't tested). I fixed this and amended my commit.